### ISSUE TITLE
Deprecate and remove the Recommendations Widget

### ISSUE THREAD

**Issue by vaurdan** [Parsely/wp-parsely#2879]
## Is your feature request related to a problem?
The Parse.ly plugin includes a Recommendation Widget, [and since WordPress 5.8 it's possible to use blocks as a widget](https://wordpress.org/documentation/article/block-based-widgets-editor/). Since this widget is pretty much the same - feature-wise - as the recommendation block, we can likely deprecate it and remove the widget.

## Describe the solution you'd like
Remove the entire `class-recommendation-widget.php` file that contains the `Recommended_Widget` class.

## Describe alternatives you've considered
If we want to deprecate the widget before actually removing it from the codebase, we could try to detect if it is in use, and recommend changing to the Recommended Block instead, and keep it for a few releases.

Alternatively, we could keep a striped down version of the recommended widget that would be a wrapper for the recommendation block - not sure how technically feasible this is, but might be worth exploring.

## Additional context
The only caveat I see from this deprecation, is users that are using the [Classic Widgets plugin](https://wordpress.org/plugins/classic-widgets/). Although, accordingly to the plugin page, the plugin "will be supported and maintained to 2024, or as long as is necessary", so eventually the ability of using classic widgets will be entirely unsupported in WordPress.

**Comment by acicovic** [comment]
I agree we should deprecate the Recommended Widget as the first step.

**Comment by thegamerjedbest45454** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
